docs/source/tutorials/mapping_reads_reference_assembly.md

```diff
@@ -0,0 +1,190 @@
+#  Mapping reads against a reference/assembly
+
+## 1. What is read-mapping?
+
+Read mapping is aligning each of your reads to a/the matching locus on a longer sequence.
+This longer sequence can either be the isolate assembly, or an assembly of a related isolate (eg. a reference isolate for the group of isolate under study).
+
+You could think of read mapping as a simulation of local hybridization of your reads to the genome, at a loci where it fits well enough to hybridize. Usually few mismatches are allowed (think about the consequences). A portion of the reads may map to several loci in your genome (ie. in repeated regions).
```

Comment:
at a locus

Reply:
Maybe add that things on the ends might not match, and maybe clipping?

Reply:
Added soft and hard clipping - please review

Reply:
ok - had corrected above, like better at